samples/cpp/throughput_benchmark/throughput_benchmark.cpp

```diff
@@ -82,6 +82,18 @@ struct Dataset {
     }
 };
 
+ov::genai::GenerationConfig greedy() {
```

Comment:
Minor: we can move such config examples to separated header shared between all samples not to duplicate code